### ISSUE TITLE
Login-state causes error when loaded with Fastrender

### ISSUE THREAD

**Issue by jmwierzbicki** [brettle/meteor-accounts-login-state#1]
I20151207-14:43:51.391(1)? Exception from task: TypeError: Cannot call method 'map' of undefined
I20151207-14:43:51.391(1)?     at PublishContext.changed (packages/meteorhacks_fast-render/lib/server/publish_context.js:54:1)
I20151207-14:43:51.392(1)?     at updateLoginStateSignedUp (packages/brettle_accounts-login-state/accounts-login-state-server.js:131:1)
I20151207-14:43:51.392(1)?     at packages/mongo/observe_multiplex.js:208:1
I20151207-14:43:51.392(1)?     at [object Object]._.extend.forEach (packages/id-map/id-map.js:48:1)
I20151207-14:43:51.392(1)?     at [object Object]._.extend._sendAdds (packages/mongo/observe_multiplex.js:200:1)
I20151207-14:43:51.392(1)?     at [object Object].kadira_Multiplexer_sendAdds [as _sendAdds](packages/meteorhacks_kadira/lib/hijack/set_labels.js:21:1)
I20151207-14:43:51.393(1)?     at packages/mongo/observe_multiplex.js:51:1
I20151207-14:43:51.393(1)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:110:1)
I20151207-14:43:51.393(1)?     at Object.task (packages/meteor/dynamics_nodejs.js:123:1)
I20151207-14:43:51.393(1)?     at [object Object]._.extend._run (packages/meteor/fiber_helpers.js:147:1)

is it possible to address this issue?

**Comment by brettle** [comment]
I just submitted kadirahq/fast-render/pull/154 which should fix this. Can you test it? Just do the following in your app dir:

```
cd packages
git clone --branch fix-142 https://github.com/brettle/fast-render.git
```

Please let me know whether that fixes it for you.

I might also be able to put together a workaround for `brettle:accounts-login-state`.

**Comment by brettle** [comment]
FYI, I was able to reproduce the problem and I just released version 0.0.4 of this package which contains a workaround for it. `meteor update` should do the trick.

**Comment by jmwierzbicki** [comment]
thank You very much, it looks like everything works like charm :) love Your work!
